.flake8

```diff
@@ -0,0 +1,25 @@
+[flake8]
+enable-extensions = G
+max-doc-length = 90
+max-line-length = 90
+select = A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,B901,B902,B903,B950
+# E226: Missing whitespace around arithmetic operators can help group things together.
```

Comment:
Maybe ignore this inline only in places where it's useful rather than disabling it globally?